dbus-cpp: 5.0.4 -> 5.0.5

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from passthru.updateScript.


meta.description for dbus-cpp is: Dbus-binding leveraging C++-11

meta.homepage for dbus-cpp is: https://gitlab.com/ubports/development/core/lib-cpp/dbus-cpp

meta.changelog for dbus-cpp is: https://gitlab.com/ubports/development/core/lib-cpp/dbus-cpp/-/blob/5.0.5/ChangeLog


###### Updates performed
- Ran passthru.UpdateScript

###### To inspect upstream changes





###### Impact

<b>Checks done</b>

---

- built on NixOS

> [!CAUTION]
> A test defined in `passthru.tests` did not pass.
- found 5.0.5 in filename of file in /nix/store/j8jqqdpysdh7n3bxx6yb478bxwx34vl5-dbus-cpp-5.0.5

---

<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
12 total rebuild path(s)

12 package rebuild(s)

First fifty rebuilds by attrpath

dbus-cpp
lomiri.biometryd
lomiri.lomiri
lomiri.lomiri-filemanager-app
lomiri.lomiri-music-app
lomiri.lomiri-session
lomiri.lomiri-system-settings
lomiri.lomiri-system-settings-security-privacy
lomiri.lomiri-system-settings-unwrapped
lomiri.mediascanner2
lomiri.trust-store
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from the cache**:
```
nix-store -r /nix/store/j8jqqdpysdh7n3bxx6yb478bxwx34vl5-dbus-cpp-5.0.5 \
  --option binary-caches 'https://cache.nixos.org/ https://nixpkgs-update-cache.nix-community.org/' \
  --option trusted-public-keys '
  nixpkgs-update-cache.nix-community.org-1:U8d6wiQecHUPJFSqHN9GSSmNkmdiFW7GW7WNAnHW0SM=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The nixpkgs-update cache is only trusted for this store-path realization.)
For the cached download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A dbus-cpp https://github.com/r-ryantm/nixpkgs/archive/94ef8b69d6b96ddae090ca066a8896d74ce64bee.tar.gz
```
Or:
```
nix build github:r-ryantm/nixpkgs/94ef8b69d6b96ddae090ca066a8896d74ce64bee#dbus-cpp
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/j8jqqdpysdh7n3bxx6yb478bxwx34vl5-dbus-cpp-5.0.5
ls -la /nix/store/j8jqqdpysdh7n3bxx6yb478bxwx34vl5-dbus-cpp-5.0.5/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review`
Commit: `94ef8b69d6b96ddae090ca066a8896d74ce64bee`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 19 packages built:</summary>
  <ul>
    <li>dbus-cpp</li>
    <li>dbus-cpp.dev</li>
    <li>dbus-cpp.doc</li>
    <li>dbus-cpp.examples</li>
    <li>lomiri.biometryd</li>
    <li>lomiri.biometryd.dev</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-music-app</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.mediascanner2</li>
    <li>lomiri.mediascanner2.dev</li>
    <li>lomiri.trust-store</li>
    <li>lomiri.trust-store.bin</li>
    <li>lomiri.trust-store.dev</li>
    <li>lomiri.trust-store.doc</li>
  </ul>
</details>

---

###### Maintainer pings

cc @OPNA2608 for [testing](https://github.com/ryantm/nixpkgs-update/blob/main/doc/nixpkgs-maintainer-faq.md#r-ryantm-opened-a-pr-for-my-package-what-do-i-do).

> [!TIP]
> As a maintainer, if your package is located under `pkgs/by-name/*`, you can comment **`@NixOS/nixpkgs-merge-bot merge`** to automatically merge this update using the [`nixpkgs-merge-bot`](https://github.com/NixOS/nixpkgs-merge-bot).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

r-ryantm committed:
dbus-cpp: 5.0.4 -> 5.0.5

OPNA2608 reviewed (commented):
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 442308`
Commit: `94ef8b69d6b96ddae090ca066a8896d74ce64bee`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 20 packages built:</summary>
  <ul>
    <li>dbus-cpp</li>
    <li>dbus-cpp.dev</li>
    <li>dbus-cpp.doc</li>
    <li>dbus-cpp.examples</li>
    <li>lomiri.biometryd</li>
    <li>lomiri.biometryd.dev</li>
    <li>lomiri.lomiri</li>
    <li>lomiri.lomiri-filemanager-app</li>
    <li>lomiri.lomiri-music-app</li>
    <li>lomiri.lomiri-session</li>
    <li>lomiri.lomiri-system-settings</li>
    <li>lomiri.lomiri-system-settings-unwrapped</li>
    <li>lomiri.lomiri-system-settings-unwrapped.dev</li>
    <li>lomiri.mediascanner2</li>
    <li>lomiri.mediascanner2.dev</li>
    <li>lomiri.trust-store</li>
    <li>lomiri.trust-store.bin</li>
    <li>lomiri.trust-store.dev</li>
    <li>lomiri.trust-store.doc</li>
    <li>nixpkgs-manual</li>
  </ul>
</details>

Most `.passthru.tests` pass.

The one on `dbus-cpp` fails because upstream forgot to bump the version number in one place. I'll add a commit to disable the version check for now and let them know about this.

`lomiri.lomiri-music-app` gets stuck, but I think this is unrelated to this bump. OCR issues again…

OPNA2608 committed:
dbus-cpp.passthru.tests.pkg-config: Disable version check for current version

Will hopefully get fixed by the next release, so do the check once version gets changed again.
